packages/react/src/Dialog/Dialog.docs.json

```diff
@@ -1,6 +1,7 @@
 {
-  "id": "drafts_dialog",
-  "name": "Dialog",
+  "id": "dialog_v2",
+  "docsId": "dialog",
+  "name": "Dialog v2",
```

Comment:
We can keep the name as `Dialog` here as well. 

The challenge with Dialog is that the story ids. Currently https://primer.style/components/dialog/react/alpha show the alpha dialog but the examples are belong to the draft dialog 😵  (If I am not mistaken) 

Can we take the dialog v2 stories under draft? and remove that `v1` from the alpha dialog to correct the story ids? I also remember there was a confusion around this and this is why we made this change, right? So we might maybe need to come up with custom solution for dialog if we don't want to revert the stories back.